### ISSUE TITLE
We rely on your plugin for real work. It's irresponsible to release untested version once in 5 months and ignore all issues.

### ISSUE THREAD

**Issue by romanr** [material-foundation/material-theme-builder#329]
It's not like we can rollback to previous version of plugin. Figma ecosystem does not have that possibility. You release broken plugin and we are stuck.  
Acting this way you affect work of thousands of designers.

**Comment by YellowPapaya** [comment]
I agree, I spent all day yesterday trying to figure out why it wasn't pulling to the neutral pallet only to realize it's a bug. And I tried to use it again now and it doesn't update at all. Why is QA not testing this properly? I would never release an app from a well know company in this state. PLEASE test your app. I lost a whole day to work yesterday.

**Comment by melcarthus** [comment]
This plugin is a nightmare. Somethimes it works, sometimes is totally does not save the colors. 
I have tried using this plugin on 2 project right now. The first project I got it working somewhat. When I open the project now the theme colors are kept when loading the theme. But in my second project no matter how many times I try to set the colors and save the theme, on every instance I re-open the project the default purple colors are reset.
